src/controllers/ArticleController.js

```diff
@@ -11,15 +11,15 @@ export default class ArticleController {
    * @returns {object} Object representing the response returned
    */
   static async saveArticle(req, res) {
-    const image = await helpers.upload(req);
+    // const image = await helpers.upload(req);
     const { tagList } = req.body;
     const newArticle = await Article.create({
       userId: req.user.id || 0,
       slug: helpers.generator.slug(req.body.title),
       title: req.body.title.trim(),
       description: req.body.description.trim(),
       body: req.body.body.trim(),
-      coverUrl: image.image.original,
+      coverUrl: req.body.coverUrl || 'placeholder.png',
```

Comment:
why have you made this update?

Reply:
I have refactored it because it was complaining in code climate, the lines were exceeding 25.

Reply:
but now since you already tested you need to remove, Gilles will update it.

Reply:
Removed